styles/pattern-library.scss

```diff
@@ -101,10 +101,7 @@ pre {
   p {
     margin: 1em;
     margin-left: 0;
-  }
-
-  & > p:first-of-type {
-    font-size: 125%;
+    font-size: 1.25rem;
```

Comment:
I bumped this for immediate readability, but it still looks a little crap. More later...